source/_integrations/traccar_server.markdown

```diff
@@ -127,6 +116,120 @@ Tracker:
 
 {% enddetails %}
 
+### Sensor - Address
+
+The Traccar Server integration will create a [sensor](/integrations/sensor) entity for each device registered in Traccar Server to show the address reported by the Traccar Server.
+
+This entity is disabled by default.
+
+{% configuration_basic %}
+Name:
+  description: The name of the sensor will be set to what you have named it in Traccar Server followed by Address.
+Entity ID:
+  description: This will be a slugified version of the name.
```

Comment:
I think an example could be helpful here, as well.

Reply:
As we can not guarantee this, I think adding that can cause more confusion.
If you have to sensors that want the same entity ID, the second one to be loaded will get a `_2` suffix.